utils/llm.py

```diff
@@ -148,10 +122,11 @@ def wrapped_model_name_matcher(model: str) -> str:
         return match[0]
     else:
         print(f"Couldn't match the modelname {model} to any known model. "
-            "Continuing but this will probably crash DocToolsLLM further "
-            "down the code.")
+              "Continuing but this will probably crash DocToolsLLM further "
```

Comment:
Ah doctoolsllm is the former name of my other project [wdoc](https://github.com/thiswillbeyourgithub/wdoc/) sorry